build/lib/accounting/write_jobs.py

```diff
@@ -0,0 +1,135 @@
+#!/usr/bin/env python3
+
+###############################################################
+# Copyright 2020 Lawrence Livermore National Security, LLC
+# (c.f. AUTHORS, NOTICE.LLNS, COPYING)
+#
+# This file is part of the Flux resource manager framework.
+# For details, see https://github.com/flux-framework.
+#
+# SPDX-License-Identifier: LGPL-3.0
+###############################################################
+from __future__ import print_function
+
+import sys
+import sqlite3
+import pandas as pd
+import time
+
+import flux.job
+import flux.constants
+import flux.util
+from flux.core.inner import raw
+
+
+def write_to_db(inactive):
+    # open connection to database
+    print("Opening JobCompletion DB...")
```

Comment:
It doesn't have to be in this PR, but I would look into converting these prints into calls to `logging`: https://docs.python.org/3/library/logging.html

Down the road, as more and more stuff is logged, it will give us fine-grained control over how much gets printed to the terminal and how much goes to a file.  There are some examples of it being used in the python front-end commands in flux-core, like `flux-mini.py`.  Unfortunately, the only examples appear to be `.error` while there are plenty of other options, like `.debug`, `.info`, and `.warn`.